infrastructure/main.tf

```diff
@@ -16,6 +16,20 @@ module "reform-notifications-db" {
   subscription       = var.subscription
 }
 
+module "reform-notifications-staging-db" {
+  source             = "git@github.com:hmcts/cnp-module-postgres?ref=master"
+  product            = "${var.product}-staging-${var.component}"
+  location           = var.location_db
+  env                = var.env
+  database_name      = "notifications-staging"
```

Comment:
db name can be same without staging suffix as the host itself would be sufficient

Reply:
can we reuse main db and just define `database_name` for staging?

Reply:
> can we reuse main db and just define `database_name` for staging?

I think not possible without changing terraform db module.

Reply:
> db name can be same without staging suffix as the host itself would be sufficient

it is sufficient, I think better to keep?

Reply:
changed the name